app/src/main/java/com/snatik/matches/common/Memory.java

```diff
@@ -3,26 +3,81 @@
 import android.content.Context;
 import android.content.SharedPreferences;
 import android.content.SharedPreferences.Editor;
+import android.preference.PreferenceManager;
+import android.util.Log;
+
+import java.util.Map;
+
+import static android.R.attr.id;
+import static android.R.id.edit;
+import static android.content.Context.MODE_PRIVATE;
 
 
 public class Memory {
 
 	private static final String SHARED_PREFERENCES_NAME = "com.snatik.matches";
 	private static String highStartKey = "theme_%d_difficulty_%d";
+	private static String bestTimeKey = "themeTime_%d_difficultyTime_%d";
 
 	public static void save(int theme, int difficulty, int stars) {
+		Log.i("In save()","");
```

Comment:
The first string in Log methods is a tag. 
`Log.i(TAG,"the text you wanna print");`

Common approach is to set const TAG and give a class name. In this case should be:
`private final static String TAG = "Memory";`

Then you can always filter logs by tag: `adb logcat -s "Memory"`